Library/Homebrew/formula.rb

```diff
@@ -405,11 +413,19 @@ def linked_keg
     Pathname.new("#{HOMEBREW_LIBRARY}/LinkedKegs/#{name}")
   end
 
+  def latest_head_prefix
+    head_prefixes = installed_prefixes.select do |pn|
+      pn.basename.to_s.start_with?("HEAD")
```

Comment:
As far as I understood it helps us to detect if `HEAD` is outdated, so we don't use it's prefix in `installed_prefix` if `head?`.
Is it true that once revision is assigned to a formula it cannot decrease or be removed even if new version is bumped?